docs/content/docs/dev/table/sql/create.md

```diff
@@ -406,6 +413,14 @@ Flink will assume correctness of the primary key by assuming that the columns nu
 
 Partition the created table by the specified columns. A directory is created for each partition if this table is used as a filesystem sink.
 
+### `DISTRIBUTED BY / DISTRIBUTED INTO`
```

Comment:
Copy the text from `SupportsBucketing` including examples.

Reply:
Ok, I copied in the text here: https://github.com/apache/flink/pull/24929/files#diff-d0ac52822e134b21138761d84268f080defc8323a120650f956afb697a1bf5f6R422-R456

How should the links to JavaDocs/specifics be handled?  (I updated some of the text; I am not sure how to line up other details.

Reply:
You can simplify the text a bit. So that it better fits to a first-level API rather then a second-level API. First-level API is used by less experienced users.